cmd/create-vm/main.go

```diff
@@ -1,52 +1,50 @@
 package main
 
 import (
-	"net/http"
+	"os"
 
 	goarg "github.com/alexflint/go-arg"
 	. "github.com/kubevirt/kubevirt-tekton-tasks/modules/create-vm/pkg/constants"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/create-vm/pkg/utils/parse"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/create-vm/pkg/vmcreator"
-	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/exit"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/log"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/output"
 	res "github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/results"
-	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/zerrors"
 	"go.uber.org/zap"
 	kubevirtv1 "kubevirt.io/api/core/v1"
 )
 
 func main() {
-	defer exit.HandleExit()
-
 	cliOptions := &parse.CLIOptions{}
 	goarg.MustParse(cliOptions)
 
 	logger := log.InitLogger(cliOptions.GetDebugLevel())
 	defer logger.Sync()
 
 	if err := cliOptions.Init(); err != nil {
-		exit.ExitOrDieFromError(InvalidCLIInputExitCode, err)
+		log.Logger().Info(err.Error())
```

Comment:
Would it make sense to log errors with a higher severity, i.e. is there `Error()`?